de.metas.document.archive/de.metas.document.archive.base/src/main/java/de/metas/document/archive/spi/impl/DocOutboundArchiveEventListener.java

```diff
@@ -63,11 +65,35 @@ private I_C_Doc_Outbound_Log createLog(final I_AD_Archive archive)
 		docExchange.setAD_Org_ID(archive.getAD_Org_ID());
 		docExchange.setAD_Table_ID(adTableId);
 		docExchange.setRecord_ID(recordId);
-		docExchange.setC_BPartner_ID(archive.getC_BPartner_ID());
-
+		
+		//
 		final int doctypeID = docActionBL.getC_DocType_ID(ctx, adTableId, recordId);
 		docExchange.setC_DocType_ID(doctypeID);
+		
+		//
+		// set isInvoiceEmailEnabled
+		final Object archiveRerencedModel = Services.get(IArchiveDAO.class).retrieveReferencedModel(archive, Object.class);
+		if (archiveRerencedModel != null)
+		{
+			final boolean isInvoiceDocument = InterfaceWrapperHelper.isInstanceOf(archive, I_C_Invoice.class);
+			final boolean isInvoiceEmailEnabled;
+			// in case of invoice document, enable email only if is enabled in partner
+			if (isInvoiceDocument)
+			{
+				final I_C_BPartner bpartner = InterfaceWrapperHelper.create(archive.getC_BPartner(), I_C_BPartner.class);
+				isInvoiceEmailEnabled = bpartner.isInvoiceEmailEnabled(); 
+			}
+			else
+			{
+				// set by defualt on Y for all other documents
+				isInvoiceEmailEnabled = true;
```

Comment:
ok..then what about adding a display rule such that the field is only shown if AD_Table_ID=<C_Invoices_Table_ID>.
i think otherwise users will percieve it as sortof unclean..
@metasnw wdyt?